README.md

```diff
@@ -68,10 +68,11 @@ The methods that run empirical models can also be exported to any pysat instrume
 
 ```
 import pysat
-import pysatMissions.methods.pyglow as mm_glow
+from pysatMissions.methods import empirical
 
 ivm = pysat.Instrument(platform='cnofs', name='ivm')
-ivm.custom.add(mm_glow.add_iri_thermal_plasma, 'modify', glat_label='glat',
+ivm.custom.add(empirical.add_iri_thermal_plasma, 'modify',
+               glat_label='glat',
```

Comment:
Should this be `attach` now?

Reply:
It should be.  I have another branch that deals with that, but I can pull those changes in here.  The big issue in terms of technical ability is how to tell a user they need pysat 3.0.0 since that is not versioned yet, and the only pip-available versions still use custom.add.

Options:
- Write everything to be pysat 3.0.0 compatible, leave a note on the front page that this is still in beta and users should use the develop-3 branch of pysat through github.
- Leave it as `.add` with a note to update later.  Maintain pip-compatibility.

Reply:
Option 3: Leave as is and note in the tutorial that this will change with pysat 3.0.0

Reply:
Report after testing: If we go with option 2 or 2.5, we have to impose more limits on this side via travis.  leaning toward option 1.